.travis.yml

```diff
@@ -21,3 +21,13 @@ script:
 
 after_script:
   - ./scripts/ci/cleanup.sh
+
+deploy:
```

Comment:
Do you want to skip cleanup ?
https://docs.travis-ci.com/user/deployment#Uploading-Files